src/binding.cc

```diff
@@ -1158,6 +1158,12 @@ MaybeBool v8__Object__HasIndex(const v8::Object& self,
       ptr_to_local(&self)->Has(ptr_to_local(&context), index));
 }
 
+MaybeBool v8__Object__HasOwnProperty(const v8::Object& self, const v8::Context& context,
+                          const v8::Value& key) {
```

Comment:
Follow the C++ API unless there are overriding concerns.

```suggestion
                          const v8::Name& key) {
```